app/controllers/api/v1/common_parameters_controller.rb

```diff
@@ -52,6 +53,17 @@ def update
       def destroy
         process_response @common_parameter.destroy
       end
+
+      def resource_class
+        GlobalLookupKey.where(:should_be_global => true)
```

Comment:
CommonParameter was turned into the default_value in GlobalLookupKey. We don't want to show all the empty values in default_value for all the GlobalLookupKeys that are created just for their overrides.